.travis.yml

```diff
@@ -1,11 +1,15 @@
 sudo: false
 language: java
-jdk:
-  - oraclejdk8
+matrix:
+  include:
+    - name: Oracle JDK 8
+      jdk: oraclejdk8
+    - name: OpenJDK 11
+      jdk: openjdk11
 cache:
   directories:
-   - $HOME/.m2
-   - $HOME/google-cloud-sdk
+    - $HOME/.m2
```

Comment:
Is our whole file formatted incorrectly? This seems to be different than that cr change you made?

Reply:
actually looks like people in general just want it to be consistent with itself

Reply:
I also got confused. Maybe the formatting in the CL is an internal style.